C#/C#.sublime-syntax

```diff
@@ -174,6 +174,15 @@ contexts:
         - include: comments
         - match: $
           pop: true
+    - match: '\b(nullable)\s+(enable|disable|restore)\s+(annotations|warnings)\b'
```

Comment:
maybe it makes sense to make the last capture group optional to avoid the need to duplicate the match pattern below without it?
```suggestion
    - match: '\b(nullable)\s+(enable|disable|restore)(?:\s+(annotations|warnings))?\b'
```

Reply:
Thanks for the suggestion! It seems to work fine.